javascript/javascript1/week1/housey-pricey.js

```diff
@@ -0,0 +1,42 @@
+let Peter = {
+    width: 8,
+    depth: 10,
+    height: 100,
+    gradenSize: 100,
+    actualCost: 2500000,
+
+}
+Peter.calculatedCost = (8 * 10 * 10 * 2.5 * 1000) + (100 * 300);
```

Comment:
nice work in here with Objects. I recommend that you use key value pairs in your calculation. It makes your code more dynamic. FX:`Peter.calculatedCost = (Peter.width * Peter.depth  etc...`

Reply:
Yes,after submitting here this point came my mind,will keep.in mind from next time,thank you.